internal/pkg/deploy/cloudformation/stack/workload.go

```diff
@@ -224,3 +236,27 @@ func envVarOutputNames(outputs []addon.Output) []string {
 	}
 	return envVars
 }
+
+// Validate that paths contain only an approved set of characters to guard against command injection.
+// We can accept 0-9A-Za-z-_.
+func validatePath(input string, maxLength int) error {
+	if len(input) > maxLength {
+		return fmt.Errorf("path must be less than %d bytes in length", maxLength)
```

Comment:
Do we want to include the path they are trying?
```suggestion
		return fmt.Errorf("path %s must be less than %d bytes in length", input, maxLength)
```

Reply:
For these validators I wanted to use a universal message. They're almost always wrapped by other methods, so the error responses from those calls include the path. Like so: https://github.com/aws/copilot-cli/pull/1956/files#diff-8852a201e2b451d8c315d78bf675f068153e17c99506a4dc6ecc692a25cb33cbR198